src/guides/v2.4/graphql/mutations/create-customer-v2.md

```diff
@@ -0,0 +1,101 @@
+---
+group: graphql
+title: createCustomerV2 mutation
+---
+
+The `createCustomerV2` mutation creates a customer account. Use the [`createCustomerAddress` mutation]({{page.baseurl}}/graphql/mutations/create-customer-address.html) to complete the customer profile and define billing and shipping addresses.
+
+The `createCustomerV2` mutation replaces the `createCustomer` mutation as the means to create a customer account. The input objects differ between these two mutations. The `createCustomer` mutation required the `CustomerInput` object, as did the `updateCustomer` mutation. The attributes required for creating a customer are different than those for updating a customer, but you could not determine this by looking at the schema. The `createCustomerV2` mutation requires the `CustomerCreateInput` object, which it does not share with the [`updateCustomerV2` mutation]({{page.baseurl}}/graphql/mutations/update-customer-v2.html).
+
+## Syntax
+
+`mutation: {createCustomerV2(input: CustomerCreateInput!) {CustomerOutput}}`
```

Comment:
Hi @keharper. In this section, we specify the `createCustomerV2` mutation but in the example below we use `createCustomer` mutation. They both use the same resolver but this part might be a little bit confusing for a reader.

Reply:
Thanks for pointing this out @rogyar . I've updated the example.